scripts/setup_linux.sh

```diff
@@ -40,7 +40,7 @@ _arg_tf_gpu="off"
 print_help ()
 {
   printf '%s\n' "Installer of garage for Linux."
-  printf 'Usage: %s [--mjkey <arg>] [--(no-)modify-bashrc] ' "$0"
+  printf 'Usage: ./scripts/setup_linux.sh [--mjkey <arg>] [--(no-)modify-bashrc] '
```

Comment:
Do not edit this part of the file. It is autogenerated (see block comments above and below).